cosmos/carbon.json

```diff
@@ -154,6 +198,17 @@
         "high": 0.17307692307692307
       }
     },
+    {
+      "coinDenom": "CANTO",
+      "coinMinimalDenom": "ibc/92E974290AF9E2BC3AEEEC35305C8FD76AC5A22A74CF8D91270FDF5A1C41E861",
+      "coinDecimals": 18,
+      "coinGeckoId": "canto",
+      "gasPriceStep": {
+        "low": 153846153846.15384,
+        "average": 192307692307.6923,
+        "high": 230769230769.23077
+      }
```

Comment:
Need to increase low step slightly, e.g. `153846153846.15385`
Refer to comment on line 187